pages/proposals/language/Espanol.json

```diff
@@ -0,0 +1,87 @@
+{
```

Comment:
Yea I will remove Spanish related code if strings can't be ready by June 14. (We need to get these live by June 15)